src/main/java/lotto/domain/LottoNumbers.java

```diff
@@ -0,0 +1,64 @@
+package lotto.domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class LottoNumbers {
+    public static final int LOTTO_NUMBERS_SIZE = 6;
+
+    private final List<LottoNumber> lottoNumbers;
+
+    public LottoNumbers(final List<LottoNumber> lottoNumbers) {
+        validateSize(lottoNumbers);
+        validateDuplicate(lottoNumbers);
+        this.lottoNumbers = lottoNumbers;
+    }
+
+    private void validateSize(final List<LottoNumber> lottoNumbers) {
+        if (Objects.isNull(lottoNumbers) || lottoNumbers.size() != LOTTO_NUMBERS_SIZE) {
+            throw new IllegalArgumentException("로또는 6개만 가능합니다.");
+        }
+    }
+
+    private void validateDuplicate(final List<LottoNumber> lottoNumbers) {
+        Set<LottoNumber> nonDuplicateLottoNumbers = new HashSet<>(lottoNumbers);
+        if (nonDuplicateLottoNumbers.size() != LOTTO_NUMBERS_SIZE) {
+            throw new IllegalArgumentException("로또 번호는 중복될 수 없습니다.");
+        }
+    }
+
+    public LottoStatus check(final LottoNumbers winningNumbers) {
+        int hitCount = 0;
+        for (LottoNumber lottoNumber : lottoNumbers) {
+            hitCount += checkWinning(winningNumbers.lottoNumbers, lottoNumber);
+        }
```

Comment:
아래와 같이 표현도 가능 하네요! :-)  
```suggestion
        int hitCount = 0;
                int hitCount = lottoNumbers.stream()
                .mapToInt(lottoNumber -> checkWinning(winningNumbers.lottoNumbers, lottoNumber))
                .sum();
```